src/pattern-library/components/patterns/hooks/UseClickAwayPage.tsx

```diff
@@ -0,0 +1,50 @@
+import Library from '../../Library';
+
+export default function UseClickAwayPage() {
+  return (
+    <Library.Page
+      title="useClickAway"
+      intro={
+        <p>
+          <code>useClickAway</code> is a hook to listen for click events on{' '}
+          <code>document.body</code>. If an event{"'"}s target is outside the
+          container element, it invokes a callback.
+        </p>
+      }
+    >
+      <Library.Section>
+        <Library.Pattern>
+          <Library.Usage symbolName="useClickAway" />
+          <Library.Example>
+            <Library.Demo
+              withSource
+              title="Basic useClickAway"
+              exampleFile="use-click-away-basic"
+            />
+          </Library.Example>
+        </Library.Pattern>
+
+        <Library.Pattern title="Hook options">
```

Comment:
This could use documentation for the positional arguments (`containerRef`, `callback`) as well.

Reply:
Right!

Reply:
Done